app/services/va_dot_gov_address_validator.rb

```diff
@@ -117,7 +117,10 @@ def ro_facility_ids_to_geomatch
     return ["vba_372"] if appeal_is_legacy_and_veteran_requested_central_office?
 
     # Return the list of RO facility IDs
-    return RegionalOffice.facility_ids if appeal.current_hearing_request_type == :travel_board
+    if appeal.current_hearing_request_type == :travel_board
+      Rails.logger.info("Travel Board Appeal Geomatching | Appeal ID: #{appeal.id}")
```

Comment:
```suggestion
      Rails.logger.info("Travel Board Appeal Geomatching | Appeal ID: #{appeal.class.name}#{appeal.id}")
```
LegacyAppeal and Appeal have overlapping ids, so we need to know which class the appeal object is.